auto_submit/test/requests/github_webhook_test.dart

```diff
@@ -6,35 +6,50 @@ import 'dart:convert';
 
 import 'package:auto_submit/requests/github_webhook.dart';
 import 'package:github/github.dart';
+import 'package:mockito/mockito.dart';
 import 'package:shelf/shelf.dart';
 import 'package:test/test.dart';
 
-import '../src/service/fake_config.dart';
 import './github_webhook_test_data.dart';
+import '../src/service/fake_config.dart';
+import '../utilities/mocks.dart';
 
 void main() {
   group('Check Webhook', () {
     late Request req;
     late GithubWebhook githubWebhook;
+    late FakeConfig config;
+    late Future<MockGithubService> futureMockGithubService;
+    final MockGithubService mockGithubService = MockGithubService();
+    late RepositorySlug slug;
+
+    const int number = 1598;
 
     setUp(() {
       req = Request('POST', Uri.parse('http://localhost/'),
           headers: {
             'header1': 'header value1',
           },
           body: webhookEventMock);
-      githubWebhook = GithubWebhook(
-        config: FakeConfig(),
-      );
+
+      Future<MockGithubService> getFuture(MockGithubService mockGithubService) async {
```

Comment:
Instead of a mock, let's use a fake.

Fake's are preferred in null safe code


```dart
class FakeGithubService extends GithubService {
  /// ...

  @override
  Future<PullRequest> getPullRequest(RepositorySlug slug, {int prNumber}) async => PullRequest(/* ... */);
}
```

Reply:
Ref: https://github.com/flutter/cocoon/blob/main/app_dart/test/src/service/fake_github_service.dart

Reply:
Thank you Keyong for the reference.